k8s/prod/cc-prod-sync-catalogue.yml

```diff
@@ -28,7 +28,8 @@ spec:
               value: "development"
             - name: GLOBAL_API_URL
               value: "https://api.citycatalyst.io/"
-            command: ["npm", "run", "sync-catalogue"]
+            command: ["/bin/sh", "-c"]
+            args: ["npx tsx scripts/catalogue-sync.ts && npx tsx scripts/emissions-factors-sync.ts"]
```

Comment:
This means the EF sync only runs if the data catalogue sync succeeds. Are we comfortable with that or should we add a separate step for it? (same with the others)